Add some more information about colors in Performance Panel (#6592)
GoogleChrome/developer.chrome.com: site/en/docs/devtools/performance/reference/index.md

`Function Call` you see that an anonymous function was called. That anonymous function then called
`Me()`, which then called `Se()`, and so on.

> DevTools assigns scripts random colors to breakup the flame chart and make it more readable. In the example above, function calls from one script are colored light green. Calls from another script are colored beige. The darker yellow represents scripting activity, and the purple event represents rendering activity. These darker yellow and purple events are consistent across all recordings.

Reviewer: ```suggestion
DevTools assigns scripts random colors to break up the flame chart and make it more readable. In the example above, function calls from one script are colored light green. Calls from another script are colored beige. The darker yellow represents scripting activity, and the purple event represents rendering activity. These darker yellow and purple events are consistent across all recordings.
```

[Long tasks are also highlighted](/blog/new-in-devtools-83/#long-tasks) with a red triangle, and with the part over 50 milliseconds shaded in red:

{% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/zB1QLni239q7xxTYqUhP.png", alt="The Long Task UI", width="800", height="568" %}

> In this example, the task is approximately 140 milliseconds so the part representing the last 90 milliseconds is shaded in red, while the initial 50 milliseconds is not.

Reviewer: ```suggestion
In this example, the task is approximately 140 milliseconds, so the part representing the last 90 milliseconds is shaded in red, while the initial 50 milliseconds is not.
```

In the example above, the **Interactions** track shows two interactions. Both have the same IDs, indicating that the interactions are triggered by the same user action.

The **Interactions** track also shows [Interaction to Next Paint (INP)](https://web.dev/inp/) warnings for interactions longer than 200 milliseconds in the **Summary** tab:

{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/wrFaZ26nYCuprtCSNB5C.png", alt="The INP warning.", width="800", height="685" %}

> Interactions over 200 milliseconds also have the part of the interaction above 200ms shaded in red—in the same way that [long tasks in the Main thread section](#http://localhost:8080/docs/devtools/performance/reference/#main:~:text=Long%20tasks%20are%20also%20highlighted) also have the part over the 50 millisecond long task threshold shaded in red. In this example 353.77 milliseconds out of the 553.77 milliseconds is shaded in red.

Reviewer: ```suggestion
Interactions over 200 milliseconds also have the part of the interaction above 200 milliseconds shaded in red—in the same way that [long tasks in the Main thread section](/docs/devtools/performance/reference/#main:~:text=Long%20tasks%20are%20also%20highlighted) also have the part over the 50 millisecond long task threshold shaded in red. In this example, 353.77 milliseconds out of the 553.77 milliseconds is shaded in red.
```
